Content.Shared/CCVar/CCVars.cs

```diff
@@ -2567,7 +2567,11 @@ public static readonly CVarDef<float>
         #region Mood System
 
         public static readonly CVarDef<bool> MoodEnabled =
+        #if RELEASE
             CVarDef.Create("mood.enabled", true, CVar.SERVER);
+        #else
+            CVarDef.Create("mood.enabled", false, CVar.SERVER);
+        #endif
```

Comment:
And now Mood doesn't work in Release configuration.

Reply:
whar

Reply:
In all instances of (!_config.GetCVar(CCVars.MoodEnabled)

Reply:
why?